Users/controllers/usersController.js

```diff
@@ -0,0 +1,63 @@
+import { usersService } from '../services';
+
+const findAllUsers = async (req, res) => {
+  try {
+    const users = await usersService.findAllUsers();
+    res.status(200).json({ users });
+  } catch (err) {
+    res.status(500).json({ message: err.message });
+  }
+};
+
+const userSignUp = async (req, res) => {
+  try {
+    const { email, name, password, account, nickname, phone_number } = req.body;
+    await usersService.userSignUp(
+      email,
+      name,
+      password,
+      account,
+      nickname,
+      phone_number
+    );
```

Comment:
```suggestion
      nickname,
      phoneNumber
    );
```

데이터베이스를 염두에 두고 snake_case로 작성해주신 것 같아요 :) 하지만 SQL문이 아닌 곳에서는 자바스크립트의 컨벤션인 camelCase를 따라주시는 것이 좋습니다 :)

Reply:
네! 수정하도록 하겠습니다.